cw_infrastructure/cw_infrastructure/clearwater_options.py

```diff
@@ -10,62 +10,63 @@
-
-        Option('scscf_uri', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.sip_uri_validator)),
-        Option('bgcf_uri', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.sip_uri_validator)),
-        Option('icscf_uri', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.sip_uri_validator)),
-
-        Option('enum_server', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.resolveable_ip_or_domain_name_list_validator)),
-        Option('signaling_dns_server', Option.OPTIONAL, vlds.ip_addr_list_validator),
-        Option('remote_cassandra_seeds', Option.OPTIONAL, vlds.ip_addr_validator),
-        Option('billing_realm', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.diameter_realm_validator)),
-        Option('node_idx', Option.OPTIONAL, vlds.integer_validator),
-        Option('ralf_hostname', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.ip_or_domain_name_with_port_validator)),
-        Option('xdms_hostname', Option.OPTIONAL,
-               vlds.run_in_sig_ns(vlds.ip_or_domain_name_with_port_validator))
+        utils.Option('hss_realm', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.diameter_realm_validator)),
+        utils.Option('hss_hostname', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.domain_name_validator)),
+        utils.Option('hs_provisioning_hostname', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.ip_or_domain_name_with_port_validator)),
+
+        utils.Option('snmp_ip', utils.Option.SUGGESTED, vlds.ip_addr_list_validator),
+        utils.Option('sas_server', utils.Option.SUGGESTED, vlds.ip_or_domain_name_validator),
+
+        utils.Option('scscf_uri', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.sip_uri_validator)),
+        utils.Option('bgcf_uri', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.sip_uri_validator)),
+        utils.Option('icscf_uri', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.sip_uri_validator)),
+
+        utils.Option('enum_server', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.resolveable_ip_or_domain_name_list_validator)),
+        utils.Option('signaling_dns_server', utils.Option.OPTIONAL, vlds.ip_addr_list_validator),
+        utils.Option('remote_cassandra_seeds', utils.Option.OPTIONAL, vlds.ip_addr_validator),
+        utils.Option('billing_realm', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.diameter_realm_validator)),
+        utils.Option('node_idx', utils.Option.OPTIONAL, vlds.integer_validator),
+        utils.Option('ralf_hostname', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.ip_or_domain_name_with_port_validator)),
+        utils.Option('xdms_hostname', utils.Option.OPTIONAL,
+                     vlds.run_in_sig_ns(vlds.ip_or_domain_name_with_port_validator)),
+
+        utils.Option('alias_list', utils.Option.DEPRECATED)
```

Comment:
I feel like `Option` is so widely used it would be useful to be able to access it without preceeding it with `utils`. You could do `from utils import Option` as well as `import utils` to do this I think.